main.tf

```
@@ -0,0 +1,117 @@
+resource "aws_vpc" "demo_vpc" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+    "Name" = "demo-vpc"
+  }
+}
+resource "aws_subnet" "public0" {
+  vpc_id     = aws_vpc.demo_vpc.id
+  cidr_block = "10.0.0.0/24"
+
+  tags = {
+    Name = "public-subnet0"
+  }
+}
+resource "aws_subnet" "public1" {
+  vpc_id     = aws_vpc.demo_vpc.id
+  cidr_block = "10.0.1.0/24"
+
+  tags = {
+    Name = "public-subnet1"
+  }
+}
+resource "aws_subnet" "private0" {
+  vpc_id     = aws_vpc.demo_vpc.id
+  cidr_block = "10.0.2.0/24"
+  tags = {
```

Nested blocks should also be separated by blank lines
```suggestion
  cidr_block = "10.0.2.0/24"
  
  tags = {
```

---

Please adjust it here and for all other nested blocks

---

Fixed it